src/net.cpp

```diff
@@ -454,52 +439,29 @@ CNode* CConnman::ConnectNode(CAddress addrConnect, const char *pszDest, bool fCo
     // Connect
     std::unique_ptr<Sock> sock;
     Proxy proxy;
-    CService addr_bind;
-    assert(!addr_bind.IsValid());
+    assert(!proxy.IsValid());
```

Comment:
In 91d97a198fb57d47870ce094454c3776bc737ba0 "net: split CConnman::ConnectNode()": why this new `assert`? If you need it all, maybe make it an `Assume`?